leasing/models/land_use_agreement.py

```diff
@@ -625,6 +630,28 @@ def __str__(self):
         )
 
 
+class LandUseAgreementReceivableType(models.Model):
+    """
+    In Finnish: Saamislaji
+    """
+
+    name = models.CharField(verbose_name=_("Name"), max_length=255)
+    sap_material_code = models.CharField(
+        verbose_name=_("SAP material code"), null=True, blank=True, max_length=255
```

Comment:
Since using `null=True` on `CharField`s and `TextField`s should be avoided, it should be removed here. (https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.Field.null).

Reply:
fixed